pkg/actuators/machine/reconciler.go

```diff
@@ -187,7 +187,30 @@ func (r *Reconciler) update() error {
 
 	klog.Infof("Updated machine %s", r.machine.Name)
 
-	r.machineScope.setProviderStatus(newestInstance, conditionSuccess())
+	vpc, err := r.awsClient.DescribeVpcs(&ec2.DescribeVpcsInput{
+		VpcIds: []*string{newestInstance.VpcId},
+	})
+	if err != nil {
+		klog.Errorf("%s: error describing vpc: %v", r.machine.Name, err)
+		return err
+	}
+
+	dhcp, err := r.awsClient.DescribeDHCPOptions(&ec2.DescribeDhcpOptionsInput{
+		DhcpOptionsIds: []*string{vpc.Vpcs[0].DhcpOptionsId},
+	})
+	if err != nil {
+		klog.Errorf("%s: error describing dhcp: %v", r.machine.Name, err)
+		return err
+	}
+	var domainName string
+	for _, i := range dhcp.DhcpOptions[0].DhcpConfigurations {
+		if *i.Key == "domain-name" {
+			klog.Infof("Domain value %v", i.Values)
```

Comment:
Do we want this to be logged? Is this debug?

Reply:
I meant to remove that, but I'm going to leave that in for now to confirm what AWS actually spits out.  I'll remove it in a few.